lib/emarsys/country.rb

```diff
@@ -0,0 +1,219 @@
+module Emarsys
+  module Country
+
+    module_function
+
+    def from_iso(iso)
+      MAPPING[iso.to_s.upcase]
```

Comment:
General utility module for mapping ISO 2-letter country codes to Emarsys proprietary internal IDs.